src/backends/github/implementation.js

```diff
@@ -89,10 +89,22 @@ export default class GitHub {
     }));
   }
 
+  getMedia() {
+    return this.api.listFiles(this.config.get('media_folder'))
+      .then(files => files.filter(file => file.type === 'file'))
+      .then(files => files.map(({ sha, name, size, download_url, path }) => {
+        return { id: sha, name, size, url: download_url, path };
+      }));
+  }
+
   persistEntry(entry, mediaFiles = [], options = {}) {
     return this.api.persistFiles(entry, mediaFiles, options);
   }
 
+  persistMedia(mediaFile, options = {}) {
+    return this.api.persistFiles(null, [mediaFile], options);
```

Comment:
Does a `null` entry still work correctly with Editorial Workflow `editorialWorkflowGit`?

Reply:
I'll check.

Reply:
Update: yes - you can test by uploading an asset with workflow enabled, always uses this method.